CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -1,28 +1,29 @@
 ==============================
 MELEE WEAPONS & UNARMED FIGHTING
 ==============================
-    These weapons/attacks use an 'Accuracy Modifier' instead of
+
+	These weapons/attacks use an 'Accuracy Modifier' instead of
 a 'Miss Chance'. This modifier is added to your combat dex modifier and
 and any relevant skill(s) to help you beat your opponent's guard DC.
 Melee weapons don't normally pierce armor, unless they state otherwise.
 
-If you have Hand-to-Hand trained, +10 damage for all melee weapons and 
+	If you have Hand-to-Hand trained, +10 damage for all melee weapons and 
 unarmed fighting. Instead, if you have Hand-to-Hand Expert, +20 damage for all 
 melee weapons and unarmed fighting. 
 
-Also, If you have melee combat skill >=20 then all DEX or STR requirements 
+	Also, If you have melee combat skill >=20 then all DEX or STR requirements 
 for melee weapons are reduced by 1.
 
-(When dividing Melee Combat Skill, drop the remainder)
+	(When dividing Melee Combat Skill, drop the remainder)
 
 ===== UNARMED FIGHTING =====
 
 == Hand-to-Hand [untrained] Free ==
 
-Damage:    5 + STR + DEX
-Range:            Adjacent (1m)
-Accuracy Modifier:  -2
-DEX Requirement:    2
+Damage: 5 + STR + DEX
+Range: Adjacent (1m)

```

Comment:
I think removing the tabs to unalign the stats is going to make this harder to read in plaintext. Personally, I don't mind, but I'm trying to advocate for 1Source in his absence.

Reply:
They weren't actually aligned or I wouldn't have attacked them. It just seemed random. (They were space characters, not tabs)